### DISCUSSION TITLE
Feature Questions

### DISCUSSION THREAD

**Discussion by JacobCoffee** [lepture/shibuya#31] [Q&A] [answered]
Any plans to add what you have in `docs/_templates/partials/globaltoc-above.html` as a more-easily configurable thing? It's pretty badass!

Also, is there a timeline on `Instant Search`? Any open issues we can help with for that?

**Comment by lepture** [comment, marked as answer]
Added a `discussion_url` theme option.

https://shibuya.lepture.com/customisation/sidebar/#discussion-url

**Comment by lepture** [comment]
Instant Search with [sphinx-docsearch](https://shibuya.lepture.com/extensions/docsearch/) is ready.

**Reply by JacobCoffee** [reply]
badass!
